src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -367,3 +369,63 @@ def count_web_vitals_snql_factory(aggregate_filter, org_id, measurement_rating,
         ],
         alias=alias,
     )
+
+
+def count_transaction_name_snql_factory(aggregate_filter, org_id, transaction_name, alias=None):
+    is_unparameterized = "is_unparameterized"
```

Comment:
usually in Python we name constants as UPPER_CASE. https://peps.python.org/pep-0008/#constants
>Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.

Reply:
Let's talk about it offline and we can resolve it in a follow up PR.